assets/src/edit-story/elements/video/controls.js

```diff
@@ -78,16 +77,27 @@ const ButtonWrapper = styled.div.attrs({ role: 'button', tabIndex: -1 })`
   }
 `;
 
-const Play = styled(PlayIcon)`
+const ButtonBackground = styled.div`
+  width: ${PLAY_BUTTON_SIZE / 2}px;
+  height: ${PLAY_BUTTON_SIZE / 2}px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #c4c4c4;
```

Comment:
Seems like we're using the hex directly here.

Reply:
We have to?

Reply:
Asked about it but will confirm again just in case.